### PR TITLE
Add --gem option to update a specific gem

### PR DESCRIPTION
**Pull request by chtitux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ instructions in the Installation section of this README.
 - Updates them each individually, using `bundle update --source #{gemname}`. To use a less
   conservative update strategy, start `bummr update` with the `--all` option.
 - Commits each gem update separately, with a commit message like:
+- Options:
+  - `--all` to include indirect dependencies (`bummr` defaults to direct dependencies only)
+  - `--group` to update only gems from a specific group (i.e. `test`, `development`)
+  - `--gem` to update only a specific gem (i.e. `tzinfo`)
```
Comment (lpender) [inline README.md]: Can line 71 (above) be moved to the line after lines 72-75 ? This is currently interrupting.

**Comment by lpender** [comment]
Thanks for the PR.

What benefit does adding this option provide vs. just having someone run 

```
bundle update #{gemname}
```

?

File: README.md (hunk 1 above)
Comment (chtitux) [reply README.md]: I have moved "- `--gem` to update only a specific gem (i.e. `tzinfo`)" to the "Options" group.

Let me know if it's not what you wanted :see_no_evil:

**Comment by chtitux** [comment]
`bummr` generates a commit message with details on the old & new versions of the gem.

It does that well, so I wanted to use it instead of manually `bundle update #{gem} && git commit -m "Update #{gem} gem from XXX to YYY"`.

**Pull request merged**